common/src/types/v0/transport/pool.rs

```diff
@@ -87,6 +88,32 @@ impl From<PoolState> for models::PoolState {
 }
 
 bus_impl_string_id!(PoolId, "ID of a pool");
+bus_impl_string_uuid!(PoolUuid, "UUID of a pool");
+
+impl TryFrom<&Value> for PoolId {
```

Comment:
Should this be `impl TryFrom<Value> for PoolId ..`

Reply:
These are not needed, we should use `Deserialize::deserialize` instead.